chsdi/lib/mortonspacekey.py

```diff
@@ -171,41 +164,47 @@ def bbox_to_morton(self, bbox):
      a) verify different strategies (3 for each base algorithm)
      b) compare performance
     '''
-
     '''
     get key for each corner point of bounding box
     return key which is common to all results (from the start)
     '''
 
     def _getCommonKey(self, keys):
         res = ''
-        # Python2/3
+        # TODO: clean-up when only Python 3.x and no longer 2.x is in use
         if not isinstance(keys, list):
             keys = list(keys)
```

Comment:
Comment and `if not isintsance(...)` are useless, if you look at the code the function is always called with a list as parameter. So remove the comment and `if`